### PR TITLE
Add isCondensed prop to the Stamp component

### PR DESCRIPTION
**Pull request by islam3zzat**
#### Summary

Currently, the stamp component has too much padding for some use-cases, the smaller version is important for use-cases that need the stamps after the text. With the small padding from the top and bottom, the stamp will be highlighted but not too much.

image

### CODE REVIEW

**Comment by Shecki** [comment]
visual it's fine for me!
Thanks

File: packages/components/stamp/src/stamp.visualroute.js [hunk 1]
```diff
@@ -24,5 +24,10 @@ export const component = () => (
     <Spec label="when secondary">
       <Stamp tone="secondary">Secondary</Stamp>
     </Spec>
+    <Spec label="when condensed">
```
Comment (tdeekens) [inline packages/components/stamp/src/stamp.visualroute.js:27]: 💯

**Review by tdeekens — APPROVED** [review]
Code looks good. Would kindly request also @jonnybel to have a look :)

**Review by jonnybel — APPROVED** [review]
Looks good 👌

File: packages/components/stamp/src/stamp.js [hunk 2]
```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { css } from '@emotion/core';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
-import InsetSquish from '@commercetools-uikit/spacings-inset-squish';
```
Comment (jonnybel) [inline packages/components/stamp/src/stamp.js:5]: Can we remove it from the package dependencies?

Comment (islam3zzat) [reply packages/components/stamp/src/stamp.js:5]: Thanks. Done https://github.com/commercetools/ui-kit/pull/1638/commits/331a4b4f288b4ef8d1f693681d43fec176e4359f

**Pull request merged**